pkg/plugin/plugin.go

```
@@ -225,6 +225,17 @@ func formatAlias(alias string, values map[string]interface{}) string {
 	return string(formattedAlias)
 }
 
+// truncateLabel shortens a legend label to at most max characters, so that a
+// long text value cannot push the rest of the panel legend off screen.
+func truncateLabel(s string, max int) string {
```

golangci-lint (2.12.2)

[error] func truncateLabel is unused

(unused)